hibernate-core/src/main/java/org/hibernate/engine/spi/LoadQueryInfluencers.java

```diff
@@ -52,12 +52,19 @@
 
 	private final EffectiveEntityGraph effectiveEntityGraph = new EffectiveEntityGraph();
 
+	private Boolean readOnly;
+
 	public LoadQueryInfluencers() {
```

Comment:
Perhaps add  `@Deprecated`

Reply:
same reason as explained below. Thanks. For the same reason I didn't change the reference of old method to new method for both make sense.